app/views/posts/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<div>
+  <h1 class="font-bold text-4xl">Posts#index</h1>
+  <p>Find me in app/views/posts/index.html.erb</p>
+</div>
```

Comment:
- [ ] As pointed above :arrow_up: , you'll equally require a view for the #show action.

Reply:
@FrederickMih Thank you for the review.
The changes have been made.